README.md

```diff
@@ -7,33 +7,42 @@ This project reads out the meter values of an [ISKRA MT681](https://wiki.volksza
 1. Install the [ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#manual-installation)
 
 2. Clone this repository:
-   ```
-   $ git clone git@github.com:christian-heusel/esp32-smartMeter.git
-   $ cd esp32-smartMeter
-   ```
 
-3. Configure the project
+   ```sh
+   git clone git@github.com:christian-heusel/esp32-smartMeter.git
+   cd esp32-smartMeter
    ```
-   $ idf.py menuconfig
+
+3. activate esp-idf in current terminal session
+
+   ```sh
+   . $HOME/esp/v5.2.1/esp-idf/export.sh
```

Comment:
The fixed version here and the missing setup instructions for the esp idf are not desirable here.
I think instead of duplicating the documentation from the esp website we should rather link it: 
https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#manual-installation

Reply:
I was following their advice and installed via vscode ide, according to
https://docs.espressif.com/projects/esp-idf/en/stable/esp32/get-started/index.html#ide

There that info was missing, which took me quite a while to figure out.